spec/controllers/api/v1/token_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+require  "#{Rails.root}/spec/support/controller_helpers.rb"
+
+describe Api::V1::TokenController, type: :controller do
+  let(:user) { create(:user) }
+  describe 'POST api/v1/request' do
+    subject { post :request_token, params: params }
```

Comment:
please don't put the exercise test step in the subject or any setup phase

Reply:
I executed the endpoint individually in every test, as in the 'contributions_controller.spec'. Is this the correct way?
ex:
```rb
it 'returns created status' do
  post :request_token, params: params
  expect(response).to have_http_status(:created)
end
```

Reply:
yep, tests are the case when you sacrifice some DRY to keep more readable, so is better to look at the test and know what is being exercised, that's why subject should be only setup too